app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,35 @@
+class OrdersController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @menus = Menu.all
+    @orders = params[:menu_id] ? Order.for_menu(params[:menu_id]) : Order.all
```

Comment:
please use explicit `present?`/`blank?` if anything acts like boolean

Reply:
also this is already looks like (a little bit) too much for an action, but still forgivable